release.py

```diff
@@ -153,19 +153,21 @@ def push_image_with_bazel(bazel_target, image_tag):
     ).format(image_tag=image_tag, target=bazel_target)
     run_or_die(command)
 
-def update_docker_images(version_tag, skip_update_latest_tag):
+def update_docker_images(version_tag, skip_update_latest_tag, update_executor_image_only, arch_specific_executor_tag):
```

Comment:
Change the update_executor_image_only to update_apps or something and flip the negation below.

Reply:
resolved offline to keep as-is because it's unclear if we added more binaries whether we'd want to built those for arm64 as well or not

Reply:
Actually I misunderstood the original intention - added individual flags for `--update_app_image`, `--update_enterprise_app_image`, and `--update_executor_image`.